selfdrive/ui/qt/widgets/ssh_keys.cc

```diff
@@ -10,10 +10,10 @@ SshControl::SshControl() : ButtonControl("SSH Keys", "", "Warning: This grants S
   hlayout->insertWidget(1, &username_label);
 
   QObject::connect(this, &ButtonControl::clicked, [=]() {
-    if (text() == "ADD") {
```

Comment:
why not use `if (text() == tr("ADD"))` ?  doesn't it work?

Reply:
Hi @deanlee,
ahh.. sorry my fault, I tested it back in 0.8.7/0.8.8 and it wasn't working, I must have forgot to translate one of the other strings, I will reverse the changes to ssh_keys :)